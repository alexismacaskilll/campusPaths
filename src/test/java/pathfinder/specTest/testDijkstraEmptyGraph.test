# Test the Find Path on empty graph.

# Create graph
CreateGraph simpleGraph

# Find shortest path from A to B
FindPath simpleGraph A B