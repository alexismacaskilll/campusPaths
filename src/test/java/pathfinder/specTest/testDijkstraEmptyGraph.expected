# Test the Find Path on empty graph.

# Create graph
created graph simpleGraph

# Find shortest path from A to B
unknown node A
unknown node B